Gemfile

```diff
@@ -27,7 +27,7 @@ gem 'link_header'
 gem 'mail'
 gem 'mailjet'
 gem 'mas-assets', git: 'git@github.com:moneyadviceservice/mas-assets'
-gem 'meta-tags', github: 'moneyadviceservice/meta-tags'
+gem 'meta-tags', github: 'kpumuk/meta-tags', branch: 'master'
```

Comment:
My plan was to ask the gem owner when the new gem will be released, so we don't have to use master.